autofs-5.1.6 - fix incorrect systemctl command syntax in autofs(8)

From: Ian Kent <raven@themaw.net>

In the body of the OPERATION section "systemctl autofs.service status"
is used when it should be "systemctl status autofs.service".

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG       |    1 +
 man/autofs.8.in |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

--- autofs-5.1.4.orig/CHANGELOG
+++ autofs-5.1.4/CHANGELOG
@@ -82,6 +82,7 @@ xx/xx/2018 autofs-5.1.5
 - fix quoted string length calc in expandsunent().
 - fix autofs mount options construction.
 - mount_nfs.c fix local rdma share not mounting.
+- fix incorrect systemctl command syntax in autofs(8).
 
 19/12/2017 autofs-5.1.4
 - fix spec file url.
--- autofs-5.1.4.orig/man/autofs.8.in
+++ autofs-5.1.4/man/autofs.8.in
@@ -58,7 +58,7 @@ service control reload action must be re
 .P
 .B @@initdir@@/autofs status
 or
-.B systemctl autofs.service status
+.B systemctl status autofs.service
 will display the status of,
 .BR automount (8),
 running or not. When using the systemd init system the status output includes
